src/models/m_edge.erl

```diff
@@ -62,6 +62,11 @@
 
 -include_lib("zotonic.hrl").
 
+-type insert_opts() :: is_insert_before
+                     | no_touch
+                     | {seq, integer()}
+                     | {creator_id, m_rsc:resource_id()}
+                     | {created, calendar:datetime()}.
```

Comment:
This type definition looks incorrect, because actually `insert_opts()` is a list. So did you mean `[insert_option()]` instead?

Reply:
Good catch. Fixed it and also renamed it to `insert_options()`